libraries/rush-lib/src/logic/PackageJsonUpdater.ts

```diff
@@ -99,6 +125,19 @@ export interface IUpdateProjectOptions {
   dependencyType?: DependencyType;
 }
 
+export interface IRemoveProjectOptions {
```

Comment:
This and `IUpdateProjectOptions` should be able to extend from a common base.

Reply:
done